tw/main.go

```diff
@@ -30,6 +31,7 @@ func main() {
 	ename := filepath.Base(os.Args[0])
 	if cmd, ok := cmds[ename]; ok {
 		if err := cmd.Execute(); err != nil {
+			clog.ErrorContextf(cmd.Context(), "%s: failed to execute command: %v", ename, err)
```

Comment:
```suggestion
			clog.Errorf("%s: failed to execute command: %v", ename, err)
```

I'm not confident but pretty sure the by the time things error, cmd.Context() is already cancelled?

Reply:
@joshrwolf since execute does not cancel context, should we remove this? 
What if we want to create a context and add some values in the context in the future clog might want to use?